package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "binary-static",
-  "version": "3.8.0",
+  "version": "3.8.1",
```

Comment:
we dont have npm module on npm for binary static, i dont think we need to bump new version on binary static now

Reply:
thanks will change to 3.8.0